infra/backup_tools/README.md

```diff
@@ -0,0 +1,10 @@
+# Backup tools
+
+The tools in this folder are designed to backup the
+{asia,eu,us}.gcr.io/k8s-artifacts-prod GCRs.
```

Comment:
If all 3 are supposed to be identical, do we need backups of all 3?  Or is it just easier that way?

Reply:
I only did it for symmetry (knee-jerk reaction).

Philosophically if I choose 1 region to back up out of the 3, then that 1 region becomes special and I don't like this.